modules/system/controllers/updates/_install_themes.htm

```diff
@@ -48,7 +48,7 @@ <h4 class="section-header">
                                 </div>
                                 <div class="details">
                                     <h4><?= $theme['name'] ?></h4>
-                                    <p>by <?= $theme['author'] ?></p>
+                                    <p><?= e(trans('cms::lang.theme.by')) ?> <?= $theme['author'] ?></p>
```

Comment:
Bad practice here. Some languages may include "by" after the author name. This should really be `by_author` with value `by :name`.